Lib/glyphsLib/cli.py

```diff
@@ -224,7 +235,7 @@ def ufo2glyphs(options):
         designspace.read(designspace_file)
         object_to_read = designspace
     elif all(source.endswith(".ufo") and os.path.isdir(source) for source in sources):
-        ufos = [defcon.Font(source) for source in sources]
+        ufos = [ufoLib2.Font.open(source) for source in sources]
```

Comment:
shouldn't this be ufo_module instead of ufoLib2?

Reply:
No because that is ufo2glyphs

Reply:
ufo_module is only for specifying the output UFO library

Reply:
Ah, right.